<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>滤波器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="PreviewCheckBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QComboBox" name="InputComboBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="InputPushButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutpuLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入图片:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ImageLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{border:1px solid rgb(0, 0, 0);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>滤波类型</string>
    </property>
    <widget class="QRadioButton" name="MedianRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>中值滤波</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="GaussianRadioButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>高斯滤波</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="OutputComboBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OutputPushButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>130</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保存目录</string>
    </property>
   </widget>
   <widget class="QLabel" name="InputLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输出路径:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
